pybamm/solvers/base_solver.py

```diff
@@ -1201,6 +1201,12 @@ def step(
         # Set up external variables and inputs
         external_variables = external_variables or {}
         inputs = inputs or {}
+        # if isinstance(inputs['Current input [A]'], pybamm.Interpolant):
+        #     del inputs['Current input [A]']
+        # elif isinstance(inputs['Voltage input [V]'], pybamm.Interpolant):
+        #     del inputs['Voltage input [V]']
+        # elif isinstance(inputs['Power input [W]'], pybamm.Interpolant):
+        #     del inputs['Power input [W]']
```

Comment:
remove?

Reply:
Actually, I realised that it is necessary but has to be fixed (will push in a sec).